tests/ssg_test_suite/test_env.py

```diff
@@ -165,11 +161,7 @@ def _delete_saved_state(self, snapshot):
         self.snapshot_stack.revert()
 
     def _local_oscap_check_base_arguments(self):
-        command_base = []
-        command_base.extend(
-            ['oscap-vm', "domain", self.domain_name,
-             'xccdf', 'eval'])
-        return command_base
+        return ['oscap-vm', "domain", self.domain_name, 'xccdf', 'eval']
```

Comment:
What if this would be just a variable instead of a function? It anyway just returns some data,

Reply:
There are multiple `_base_arguments` methods. I agree these could be simplified. As the fix is piggybacked just because it was blocking, I would prefer not to rework it more than necessary. Can be adjusted in different PR.